cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -373,11 +375,11 @@ data:
       # Set queue_full action to block because we want to pause gracefully
       # in case of the off-the-limits load instead of throwing an exception
       buffer_queue_full_action block
-      # Set the chunk limit conservatively to avoid exceeding the GCL limit
-      # of 10MiB per write request.
-      buffer_chunk_limit 2M
+      # Set the chunk limit conservatively to avoid exceeding the Stackdriver
+      # Logging limit of 5MiB per write request.
```

Comment:
5MiB is not a limit, it's the recommended size. See the comment I referenced earlier (https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/templates/etc/td-agent/td-agent.conf#L18)...

Reply:
Oh, from reading https://cloud.google.com/logging/quotas I had the impressions it is a limit

I'll fix the comment

Reply:
Thanks!